dev/tests/integration/framework/Magento/TestFramework/Helper/Amqp.php

```diff
@@ -13,26 +13,57 @@
  */
 class Amqp
 {
+    const CONFIG_PATH_QUEUE_AMQP_HOST = 'queue/amqp/host';
+    const CONFIG_PATH_QUEUE_AMQP_USER = 'queue/amqp/user';
+    const CONFIG_PATH_QUEUE_AMQP_PASSWORD = 'queue/amqp/password';
```

Comment:
I believe shorter versions like `CONFIG_PATH_PASSWORD` would be enough taking into account `Amqp` class context.